.github/workflows/build_and_test.yml

```diff
@@ -488,7 +489,7 @@ jobs:
         run: echo "{\"credsStore\":\"ecr-login\"}" > /kaniko/.docker/config.json
 
       - name: Kaniko build compute node with extensions
-        run: /kaniko/executor --snapshotMode=redo --cache=true --cache-repo 369495373322.dkr.ecr.eu-central-1.amazonaws.com/cache --snapshotMode=redo --context . --dockerfile Dockerfile.compute-node --destination 369495373322.dkr.ecr.eu-central-1.amazonaws.com/compute-node:$GITHUB_RUN_ID
+        run: /kaniko/executor --snapshotMode=redo --cache=true --cache-repo 369495373322.dkr.ecr.eu-central-1.amazonaws.com/cache --snapshotMode=redo --context . --dockerfile Dockerfile.compute-node --build-arg "PG_VERSION='${{matrix.postgres_version}}' --destination 369495373322.dkr.ecr.eu-central-1.amazonaws.com/compute-node:$GITHUB_RUN_ID-${{matrix.postgres_version}}
```

Comment:
It would be better to have separate ECR repositories per version, I would suggest to add `compute-node-15` here: https://github.com/neondatabase/build/blob/main/terraform/environment/dev/main.tf

Reply:
Can you please, explain what is the difference between these approaches?

Should I also rename `ecr-compute-node` to `ecr-compute-node-v14` ?

Reply:
Currently image building will result in: `compute-node:000000-14` - or `compute-node:000000-15`- this can cause confusion even after they receive a `latest` tag in addition.

Renaming to make the naming setup consistent will help keeping pipelines tidy

Reply:
Yes that is the correct approach

Reply:
Crane copies the images over to Docker Hub, ECR source shouldn't be a problem here - https://github.com/neondatabase/neon/blob/main/.github/workflows/build_and_test.yml#L548

Reply:
`cloud` repo depends on these containers too. 
To make the deploy seamless, I'll add caching to both old and new repos:
```
        run: /kaniko/executor --snapshotMode=redo --cache=true --cache-repo 369495373322.dkr.ecr.eu-central-1.amazonaws.com/cache --snapshotMode=redo --context . --dockerfile Dockerfile.compute-node --build-arg "PG_VERSION=v14' --destination 369495373322.dkr.ecr.eu-central-1.amazonaws.com/compute-node:$GITHUB_RUN_ID
        
        run: /kaniko/executor --snapshotMode=redo --cache=true --cache-repo 369495373322.dkr.ecr.eu-central-1.amazonaws.com/cache --snapshotMode=redo --context . --dockerfile Dockerfile.compute-node --build-arg "PG_VERSION=v14' --destination 369495373322.dkr.ecr.eu-central-1.amazonaws.com/compute-node-14:$GITHUB_RUN_ID
        
        run: /kaniko/executor --snapshotMode=redo --cache=true --cache-repo 369495373322.dkr.ecr.eu-central-1.amazonaws.com/cache --snapshotMode=redo --context . --dockerfile Dockerfile.compute-node --build-arg "PG_VERSION=v15' --destination 369495373322.dkr.ecr.eu-central-1.amazonaws.com/compute-node-15:$GITHUB_RUN_ID
```
after that switch all `compute-node` paths in `cloud` repo
and after that we can remove `compute-node` from here and from `build` repo.

Is there anything else that this PR can break in deploy process?

Reply:
Kaniko caching should take care of the identical images, we can keep it running side by side until the other repo's are updated to use the new images

Reply:
@zoete 

Is there any difference between `compute-node-14:latest` and `compute-node:latest-14` except for readability (which, IMHO doesn't differ too much...)?

Reply:
We can do a single ECR repository, just make sure everybody is aware of the tagging approach